src/silo/query_engine/query_engine.cpp

```diff
@@ -7,21 +7,14 @@
 
 // query_parse_exception.h must be before the RAPIDJSON_ASSERT because it is used there
 #include "silo/query_engine/query_parse_exception.h"
```

Comment:
Please also delete that comment now and move the include in line with the other SILO includes.